tests/test_catalog.py

```diff
@@ -447,10 +448,16 @@ def test_construct_order_parameters_live(catalog_live, product_id):
 
 
 # pylint: disable=unused-argument
-def test_estimate_order_from_catalog(order_parameters, order_mock, catalog_mock, requests_mock):
-    url_order_estimation = f"{API_HOST}/workspaces/" f"{catalog_mock.auth.workspace_id}/orders/estimate"
-    requests_mock.post(url=url_order_estimation, json={"data": {"credits": 100}})
-    estimation = catalog_mock.estimate_order(order_parameters)
+def test_estimate_order_from_catalog(order_parameters, requests_mock, auth_mock):
```

Comment:
The endpoint supports both order types. The function has a lot of references to "catalog" though. Will it support tasking orders as well?

Reply:
For now, the idea is to keep working as before, not supporting tasking orders yet.